### PR TITLE
Pipelines maintain changes after review session

### PR DESCRIPTION
**Pull request by MoonMoon1919**
Changes to maintaining pipelines after review from Eben

### CODE REVIEW

File: _docs-sources/pipelines/maintain/extending.md [hunk 1]
```diff
@@ -50,27 +61,36 @@ infrastructure-deployer --aws-region "us-east-1" -- docker-image-builder build-d
...
 
-## Adding script arguments
+## Adding scripts that can be run in Pipelines
 
 The `deploy-runner` Docker image for Pipelines only allows scripts within a single directory to be executed in the ECS task as an additional security measure.
 
-By default, the `deploy-runner` ships with three scripts - one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
+By default, the `deploy-runner` ships with three scripts — one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
 
 If you need to run a custom script in the `deploy-runner`, you must fork the image code, add an additional line to copy your script into directory designated by the `trigger_directory` argument. Then, you will need to rebuild the Docker image, push to ECR, then update your Pipelines deployment following the steps in [Updating Pipelines](./updating.md).
```
Comment (oredavids) [inline _docs-sources/pipelines/maintain/extending.md:96]: Seems like a good candidate for wrapping in an admonition for visual callout.

File: _docs-sources/pipelines/maintain/updating.md [hunk 2]
```diff
@@ -17,7 +20,7 @@ This guide assumes you have the following:
 
 ## Updating container images
 
-Gruntwork Pipelines uses two images - one for the [Deploy Runner](https://github.com/gruntwork-io/terraform-aws-ci/blob/main/modules/ecs-deploy-runner/docker/deploy-runner/Dockerfile) and one for [Kaniko](https://github.com/gruntwork-io/terraform-aws-ci/blob/main/modules/ecs-deploy-runner/docker/kaniko/Dockerfile). To update pipelines to the latest version, you must build and push new versions of each image.
+Gruntwork Pipelines uses two images — one for the [Deploy Runner](https://github.com/gruntwork-io/terraform-aws-ci/blob/main/modules/ecs-deploy-runner/docker/deploy-runner/Dockerfile) and one for [Kaniko](https://github.com/gruntwork-io/terraform-aws-ci/blob/main/modules/ecs-deploy-runner/docker/kaniko/Dockerfile). To update pipelines to the latest version, you must build and push new versions of each image.
```
Comment (oredavids) [inline _docs-sources/pipelines/maintain/updating.md:23]: Nit: use unordered list.

File: _docs-sources/pipelines/maintain/updating.md [hunk 3]
```diff
@@ -83,14 +87,16 @@ aws-vault exec your-stage -- terragrunt apply --terragrunt-source-update -auto-a
...
+If you’ve deployed Pipelines as a standalone framework using the `ecs-deploy-runner` service in the Service Catalog, refer to the [Variable Reference](../../reference/services/ci-cd-pipeline/ecs-deploy-runner#reference) section for the service in the Library Reference for configuration details. You will need to update the `docker_tag` value in the `container_image` object for the [ami_builder_config](../../reference/services/ci-cd-pipeline/ecs-deploy-runner#ami_builder_config), [docker_image_builder_config](../../reference/services/ci-cd-pipeline/ecs-deploy-runner#docker_image_builder_config), [terraform_applier_config](../../reference/services/ci-cd-pipeline/ecs-deploy-runner#terraform_applier_config), and [terraform_planner_config](../../reference/services/ci-cd-pipeline/ecs-deploy-runner#terraform_planner_config) variables.
```
Comment (oredavids) [inline _docs-sources/pipelines/maintain/updating.md:93]: Nit: use unordered list.

File: _docs-sources/pipelines/maintain/extending.md [hunk 4]
```diff
@@ -50,27 +61,36 @@ infrastructure-deployer --aws-region "us-east-1" -- docker-image-builder build-d
...
+</TabItem>
+</Tabs>
 
-## Adding script arguments
+## Adding scripts that can be run in Pipelines
 
 The `deploy-runner` Docker image for Pipelines only allows scripts within a single directory to be executed in the ECS task as an additional security measure.
 
-By default, the `deploy-runner` ships with three scripts - one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
+By default, the `deploy-runner` ships with three scripts — one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
```
Comment (oredavids) [inline _docs-sources/pipelines/maintain/extending.md:93]: Nit: use unordered list.

Comment (MoonMoon1919) [reply _docs-sources/pipelines/maintain/extending.md:93]: @oredavids I discussed the unordered list vs in paragraph approach live. In these cases Eben was okay with the in paragraph approach.

File: _docs-sources/pipelines/maintain/extending.md (hunk 1 above)
Comment (MoonMoon1919) [reply _docs-sources/pipelines/maintain/extending.md:96]: In this case I think putting this in an admonition might draw the eye away from the first paragraphs that set up context. WDYT?

Comment (oredavids) [reply _docs-sources/pipelines/maintain/extending.md:96]: That makes sense. I am okay with leaving it as it is.

**Pull request merged**